src/dmd/dtemplate.d

```diff
@@ -8288,46 +8288,108 @@ struct TemplateStats
     uint numInstantiations;     // number of instantiations of the template
     uint uniqueInstantiations;  // number of unique instantiations of the template
 
+    TemplateInstances* uniqueInstances;
+
     /*******************************
      * Add this instance
      */
-    static void incInstance(const TemplateDeclaration td)
+    static void incInstance(const TemplateDeclaration td,
+                            const TemplateInstance ti)
     {
         if (!global.params.vtemplates)
             return;
         if (!td)
             return;
+        if (!ti)
+            return;
         if (auto ts = cast(const void*) td in stats)
+        {
```

Comment:
@MoonlightSentinel Shall I remove these braces aswell?

Reply:
Yes, their addition doesn't provide any value for this PR.